mlir/include/mlir/Target/LLVM/ModuleToObject.h

```diff
@@ -23,6 +23,13 @@ class TargetMachine;
 
 namespace mlir {
 namespace LLVM {
+
+/// Translate LLVM module to textual ISA.
+FailureOr<std::string>
```

Comment:
Not blocking on this, but if you have time. Can we move to SmallString?

Reply:
Ok, I tried it and it requires a more substantial changes in callbacks interfaces, I will add a TODO leave it for the future PRs